lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
+      color: var(--nx-color-vulnerability-#{$severity}-text);
+      border: 1px solid var(--nx-color-vulnerability-#{$severity}-border);
+    }
+  }
+}
+
+.nx-small-vulnerability-counter {
+  $height: 25px;
```

Comment:
This value does't match the spec.  The counters appear to only be 21px tall.  Additionally you might not need to set the `height` explicitly, it'd be better to see if you can get it to just work out naturally with a careful choice of line-height.

Reply:
> This value does't match the spec. The counters appear to only be 21px tall. Additionally you might not need to set the `height` explicitly, it'd be better to see if you can get it to just work out naturally with a careful choice of line-height. Be sure to take the borders into account - with your current styles the counters are actually _27px_ tall.

Thanks for pointing that out, I've fixed it here: [64ac0b0](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/64ac0b040342a05a80ee381e5c241da54e06b0c2)